osc/cmdln.py

```diff
@@ -788,10 +788,10 @@ def _help_get_command_list(self):
             # practice dictates that command help strings begin with this, but
             # it isn't at all wanted for the command list.
             to_strip = "${cmd_name}:"
-            if doc and doc.startswith(to_strip):
+            if doc and doc.lstrip().startswith(to_strip):
                 #log.debug("stripping %r from start of %s's help string",
                 #          to_strip, cmdname)
-                doc = doc[len(to_strip):].lstrip()
+                doc = (doc.lstrip())[len(to_strip):].lstrip()
```

Comment:
Wouldn't be better to do `doc = doc.lstrip()` before `if doc ...`?
It is more efficient (running `lstrip()` only once) and also covers a situation (unlikely one) that the docstring contains only whitespaces.